operator/model/src/main/java/io/apicurio/registry/operator/api/v3/spec/Configuration.java

```diff
@@ -0,0 +1,36 @@
+package io.apicurio.registry.operator.api.v3.spec;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+
+import java.util.List;
+
+@JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "env" })
+@JsonDeserialize(using = JsonDeserializer.None.class)
+public class Configuration implements KubernetesResource {
```

Comment:
Why not use Lombok? (Also elsewhere). All fabric8 models use Lombok and it's far more readable.